homeassistant/components/cert_expiry/sensor.py

```diff
@@ -152,10 +158,14 @@ async def async_update(self):
 
         self._available = True
         self._error = None
-        self._state = days_to_expiry
+        expiry = self._timestamp - dt.now()
+        self._state = expiry.days
         self._valid = True
 
     @property
     def device_state_attributes(self):
         """Return additional sensor state attributes."""
-        return {"is_valid": self._valid, "error": str(self._error)}
+        attributes = {"is_valid": self._valid, "error": str(self._error)}
+        if self._timestamp:
+            attributes["expires_at"] = self._timestamp
```

Comment:
I think we should make a separate timestamp sensor with device class timestamp.

https://developers.home-assistant.io/docs/core/entity/sensor#available-device-classes

Reply:
To be clear, expose two sensor entities per certificate?

Reply:
Yeah, if we want to be backwards compatible. Ideally we should just have one timestamp sensor, but that's a breaking change.

Reply:
Refactored to use `DataUpdateCoordinator` and to expose a separate sensor for the timestamp.